RELEASE/scripts/autoscend/auto_post_adv.ash

```diff
@@ -642,19 +642,35 @@ void handlePostAdventure()
 		}
 
 		boolean doML = true;
+		boolean removeML = false;
```

Comment:
This may be a good opportunity to do #56 and move this stuff from post combat to precombat. I dont think it would be a ton of work but i havent looked at how engrained it is in post. Ill leave it up to you if you want to do it or not.

Reply:
I don't know what this would break. Even Jasper wasn't sure when he submitted that ticket. I don't want to undertake pulling it out and risking breaking the script.